opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1084,6 +1084,7 @@ namespace Opm {
 
         const Opm::SummaryConfig& summaryConfig = ebosSimulator_.vanguard().summaryConfig();
         const auto& summaryState = ebosSimulator_.vanguard().summaryState();
+        const bool write_restart_file = ebosSimulator_.vanguard().eclipseState().getRestartConfig().getWriteRestartFile(reportStepIdx);
```

Comment:
I am testing this, while please correct `eclipseState` to be `eclState` .

Reply:
:+1: